test/parallel/test-pipe-writev.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const net = require('net');
+
+if (common.isWindows) {
+  common.skip('Unix-specific test');
+  return;
+}
+
+common.refreshTmpDir();
+
+const server = net.createServer((connection) => {
+  connection.on('error', (err) => {
+    throw err;
+  });
+
+  const writev = connection._writev.bind(connection);
+  connection._writev = common.mustCall(writev);
+
+  connection.cork();
+  connection.write('pi');
+  setTimeout(() => {
+    connection.write('ng');
+    connection.end();
+  }, 200);
```

Comment:
So I tested it and yeah, even `setTimeout` is useless here, not to mention `process.nextTick`. I'll remove it completely. Though let it be two writes, just in case.